### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by tschuliiia**
seperated examples and explaination

### CODE REVIEW

File: docs/guide/getstarted/configuration.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ tags: ['tag1','tag2'],  // specify some tags (e.g. if you use Sauce Labs)
 name: 'my test'         // set name for test (e.g. if you use Sauce Labs)
 ```
 
+####desiredCapabilities for Appium
+If you want to reuse your tests of WebdriverIO together with Appium for real device tests, you just need to exchange the port at the desiredCapabilities and add some options. Android tests are possible with Chrome, iOS tests with Safari, but you need the SafariLauncher to start the Safari. It need to be installed on the device and opened as a native app. It can be downloaded here : https://github.com/budhash/SafariLauncher
+Appium also provides native App testing, but in this case, some App data are needed.
```
Comment (christian-bromann) [inline docs/guide/getstarted/configuration.md]: This is not quite true. You can run Appium tests without providing an app

Comment (tschuliiia) [inline docs/guide/getstarted/configuration.md]: What do you mean without proving an app? just the browser testing with appium?

Comment (christian-bromann) [inline docs/guide/getstarted/configuration.md]: yeah, browser testing on a mobile device though, for example with these capabilities:

```js
caps['browserName'] = 'Safari';
caps['appiumVersion'] = '1.4.10';
caps['deviceName'] = 'iPhone 6 Plus';
caps['deviceOrientation'] = 'portrait';
caps['platformVersion'] = '8.4';
caps['platformName'] = 'iOS';
```

**Comment by christian-bromann** [comment]
I guess we can close this PR, as I don't thing that this paragraph adds much value to the docs.

**Pull request closed without merging**